docs/external-pipeline/platform_upload_buckit_messages.md

```diff
@@ -85,7 +85,12 @@ by the external data pipeline:
   "category": "test",
   "metadata": {
     "reporter": "",
-    "stale_timestamp": "0001-01-01T00:00:00Z"
+    "stale_timestamp": "0001-01-01T00:00:00Z",
```

Comment:
that's **quite** stale :)

Reply:
Well, I just added a comma. Normally it will contain the "Now" when it was processed by Ingress